### PR TITLE
Pack strings

### PR DESCRIPTION
**Pull request by ziggi**
Also in this PR:
- Removed GetPlayerSex
- format changed to strcat (faster)

### CODE REVIEW

File: skintags.inc [hunk 1]
```diff
@@ -129,384 +129,388 @@
...
+    {!"crogrl3", !"Millie Perkins"                                     , SKIN_GENDER_FEMALE | SKIN_RACE_WHITE},
+    {!"gangrl3", !"Denise Robinson"                                    , SKIN_GENDER_FEMALE | SKIN_RACE_BLACK},
+    {!"cwfofr" , !"Farm-Town inhabitant"                               , SKIN_GENDER_FEMALE | SKIN_RACE_WHITE | SKIN_TYPE_FARMER},
+    {!"cwfohb" , !"Hillbilly"                                          , SKIN_GENDER_FEMALE | SKIN_RACE_WHITE | SKIN_TYPE_FARMER},
+    {!"cwfyfr1", !"Farm-Town inhabitant"                               , SKIN_GENDER_FEMALE | SKIN_RACE_WHITE | SKIN_TYPE_FARMER},
+    {!"cwfyfr2", !"Farm-Town inhabitant"                               , SKIN_GENDER_FEMALE | SKIN_RACE_WHITE | SKIN_TYPE_FARMER},
+    {!"cwmyhb2", !"Hillbilly"                                          , SKIN_GENDER_MALE   | SKIN_RACE_WHITE | SKIN_TYPE_FARMER},
+    {!"dwfylc2", !"Farmer"                                             , SKIN_GENDER_FEMALE | SKIN_RACE_WHITE | SKIN_TYPE_FARMER},
+    {!"dwmylc2", !"Farmer"                                             , SKIN_GENDER_MALE   | SKIN_RACE_WHITE | SKIN_TYPE_FARMER
```
Comment (Crayder) [inline skintags.inc:364]: Why remove it? It's more general than "gender" in some places.

Comment (Crayder) [inline skintags.inc:364]: Ternary is more my style for this.

**Comment by Crayder** [comment]
Add back `GetPlayerSex` ("sex" is more general than "gender" in some areas)

And do this for `GetSkinGenderName`

``` pawn
stock GetSkinGenderName(skinid, output[], outlen = sizeof(output)) { 
    output[0] = '\0';
    strcat(output, GetSkinGender(skinid) ? ("male") : ("female"), outlen);
}
```

Then I'll pull. I also need to do some stuff, so hurry please... :D

**Comment by ziggi** [comment]
> Add back GetPlayerSex ("sex" is more general than "gender" in some areas)

Are you sure? Because right name for your GetPlayerSex is a GetSkinSex.
GetPlayerSex should be like:

``` Pawn
#define GetPlayerSex(%0) (GetSkinGender(GetPlayerSkin(%0)))
```

> And do this for GetSkinGenderName

Done.

**Comment by Crayder** [comment]
Lol, yeah, I meant `GetSkinSex`. I can do that myself though, thanks!

**Pull request merged**